Gds/src/fprime_gds/executables/fprime_cli.py

```diff
@@ -96,7 +99,8 @@ def add_search_arguments(parser: argparse.ArgumentParser, command_name: str):
         "--components",
         nargs="+",
         type=str,
-        help='only show %s from the given component name "COMP"; can provide multiple components to show %s from all components given'
+        help='only show %s from the given component name "COMP"; can provide multiple'
+             'components to show %s from all components given'
```

Comment:
Same as above. I prefer to not separate the strings into two quotes on separate line. Might be better to revert it back to what they were.